plugins/SitesManager/templates/_siteWithoutDataTabs.twig

```diff
@@ -90,11 +90,15 @@
 
         <p>{{ 'CoreAdminHome_JSTracking_CodeNoteBeforeClosingHead'|translate("&lt;/head&gt;")|raw }}</p>
 
-        <pre vue-directive="CoreHome.SelectOnFocus">{{ jsTag|raw }}</pre>
+        <div
+                vue-entry="CoreAdminHome.JsTrackingCodeGeneratorSitesWithoutData"
+                default-site="{{ defaultSiteDecoded|json_encode }}"
+                max-custom-variables="{{ maxCustomVariables|json_encode }}"
+                server-side-do-not-track-enabled="{{ serverSideDoNotTrackEnabled|json_encode }}"
```

Comment:
Do we need to `json_encode` a boolean and a number here?

Reply:
I have no idea what is the correct way here

Reply:
@sgiehl what's your recommendation here?

Reply:
@michalkleiner  `json_encode` is used like this when passing parameters for most of our vue components. Whether this is ideal or not, it is not a pattern being introduced by this PR but the continuation of an existing practice, so I think it is okay to do this here for consistency and we can discuss other approaches separately if needed.